automation.md

```diff
@@ -0,0 +1,102 @@
+![background](img/logo.png)
+
+[Clicknium](https://www.clicknium.com/) offers automation technology for a variety of uses. It can handle several forms of automation, including SAP automation, Java automation, Windows desktop automation, and image automation. Let's discuss about the automation technology that underlies.
+
+# Introduction
+
+Regarding software automation technology, it was mainly used for software testing at the earliest, especially when it comes to UI automation testing. Many system software designs take into account how to make it accessible to people with disabilities. See [windows accessibility](https://www.microsoft.com/en-us/accessibility/?rtc=1). In recent years, RPA (Robotic Process Automation) has been accepted by more and more enterprises. RPA can play a great role in multiple business field scenarios. process, to reduce costs and increase efficiency for enterprises, and quickly improve the level of informatization. RPA products generally include automated operation components for various software, data processing components, process logic control components, etc., of which UI automation components are the most used.
```

Comment:
Regarding software automation technology, it was mainly used for software testing in the very begining, especially for UI automation testing.

Reply:
Many system software designs take the accessibility to the disabled users into account.

Reply:
RPA plays a great role in multiple business field scenarios to optimize the process,  reduce costs and increase efficiency, and quickly improve the level of the informatization.

Reply:
RPA products generally include automated operation components for various softwares, data processing components, process logic control components, etc., among which UI automation components are the mostly used.